packages/mobile/src/helpers/date/get-strict-time-to-event/get-strict-time-to-event.ts

```diff
@@ -0,0 +1,28 @@
+import dayjs from 'dayjs';
+import { DateTimeFormat } from '~/common/enums/enums';
+
+const getStrictTimeToEvent = (date: Date) => {
+  const duration = dayjs.duration(dayjs(date).diff(new Date()));
+  const totalMs = duration.asMilliseconds();
+  const [days, hours, minutes, seconds] = duration
+    .format(DateTimeFormat.DD_HH_mm_ss)
+    .split('-');
```

Comment:
Please take a look: https://day.js.org/docs/en/durations/get

Reply:
I didn't find any method which returns '09', '08', '07' ... instead of 9, 8, 7 for any specific type of time